app/main/views.py

```diff
@@ -517,3 +520,116 @@ def mousereport_pdf(animal_id):
+
+
+@main.route('/surgery/status', methods=['GET', 'POST'])
+def surgery_status():
+    fexperiment = dj.create_virtual_module("csmith_testing", "csmith_testing")
+    form = forms.SurgeryStatusForm(request.form)
+    surgery_data = (fexperiment.Surgery & {'surgery_outcome': 'Survival'}).fetch(as_dict=True)
+    newest_status = []
+    for entry in surgery_data:
+        status_key = dict((k,entry[k]) for k in ('animal_id','date') if k in entry)
+        if len(fexperiment.SurgeryStatus & status_key) > 0:
+            if (datetime.date.today() - entry['date']).days < 5:
+                newest_status.append((fexperiment.SurgeryStatus & status_key).fetch(order_by="timestamp DESC")[0])
+    table = tables.SurgeryStatusTable(newest_status)
+    return render_template('surgery_status.html', form=form, table=table)
+
+
+@main.route('/surgery/update/<animal_id>/<date>', methods=['GET', 'POST'])
+def surgery_update(animal_id, date):
+    key = {'animal_id': animal_id, 'date': date}
+    fexperiment = dj.create_virtual_module('csmith_testing', 'csmith_testing')
+    form = forms.SurgeryEditStatusForm(request.form)
+    if request.method == 'POST':
+        tuple_ = {'animal_id': form['animal'].data, 'date': str(form['date_field'].data),
+                  'day_one': int(form['dayone_check'].data), 'day_two': int(form['daytwo_check'].data),
+                  'day_three': int(form['daythree_check'].data),
+                  'euthanized': int(form['euthanized_check'].data), 'checkup_notes': form['notes'].data}
+        try:
+            fexperiment.SurgeryStatus.insert1(tuple_)
+            flash("Surgery status for animal {} on date {} updated.".format(animal_id, date, tuple_['day_one']))
+        except Exception as ex:
+            ex_message = "An exception of type {} occurred. More information:\n\n{}".format(type(ex).__name__,
+                                                                                            ex.args)
+            flash(ex_message)
+            flash(tuple_)
+        return redirect(url_for('main.surgery_status'))
+    if len(fexperiment.SurgeryStatus & key) > 0:
+        data = (fexperiment.SurgeryStatus & key).fetch(order_by='timestamp DESC')[0]
+        return render_template('surgery_edit_status.html', form=form, animal_id=data['animal_id'],
+                               date=data['date'], day_one=bool(data['day_one']), day_two=bool(data['day_two']),
+                               day_three=bool(data['day_three']), euthanized=bool(data['euthanized']),
+                               notes=data['checkup_notes'])
+    else:
+        return render_template('404.html')
+
+
+@main.route('/api/v1/surgery/notification', methods=['GET'])
+def surgery_notification():
+    fexperiment = dj.create_virtual_module('csmith_testing', 'csmith_testing')
+    num_to_word = {1: 'one', 2: 'two', 3: 'three'}
+    slack_notification_channel = "#slack_api_testing"
+    slack_manager = "cameron.smith"
```

Comment:
Are you gonna keep receiving this?

Reply:
This will change before the final commit. I don't want anyone getting unnecessary notifications during debugging.